public/css/dashboard.css

```diff
@@ -438,12 +493,6 @@ body {
                 "main main";
     }
 
-    body.no-title {
```

Comment:
Why remove this

Reply:
I've moved body.no-title closer to body declaration, it makes more sense to me as it helps to see the difference in the grid

Reply:
Don't do that in this PR, separate functional changes from reformatting in a separate PR.

It's fine with me if you want to reorganize the whole file but don't do that in a PR that also changes in the contents functionally.

For now this is okay; but it just takes me a whole lot longer to identify what's going on.

--> side note, `.no-title` view is broken for the dashboard; but that's a different issue.

Reply:
Ok, I understand, thanks